### PR TITLE
DBT-712 Feat/release staging

### PR DESCRIPTION
**Pull request by abelbordonado**
## Context

Made some changes on the migrations (were broken and I couldn't generate them) 
On the README and package.json.

The goal was to be able to generate a new instance in the staging server

> [Staging Server](https://academy750.tianlu.tech/)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,59 +1,69 @@
...
+
+```
+mysql -u root
+CREATE DATABASE bomberosapimysql;
+CREATE DATABASE bomberos_test;
+CREATE USER 'bomberos'@'localhost' IDENTIFIED WITH mysql_native_password BY '123456';
+GRANT ALL ON `bomberosapimysql`.* TO `bomberos`@`localhost`;
+GRANT ALL ON `bomberos_test`.* TO `mindkind`@`localhost`;
+
+USE mindkind
```
Comment (kayiganwa04) [inline README.md:47]: USE mindkind 🐞

**Review by kayiganwa04 — CHANGES_REQUESTED** [review]

File: README.md [hunk 2]
```diff
@@ -1,59 +1,69 @@
...
+
+3 - Create a user and grand him access 
+
+```
+mysql -u root
+CREATE DATABASE bomberosapimysql;
+CREATE DATABASE bomberos_test;
+CREATE USER 'bomberos'@'localhost' IDENTIFIED WITH mysql_native_password BY '123456';
+GRANT ALL ON `bomberosapimysql`.* TO `bomberos`@`localhost`;
+GRANT ALL ON `bomberos_test`.* TO `mindkind`@`localhost`;
```
[the remark this answers is not in the document]
Comment (abelbordonado) [reply README.md:45]: Damm good comment

**Review by kayiganwa04 — APPROVED** [review]
One question how about the issues of not getting the password in the email? you remember for us we had to edit student password in the database manually

**Comment by abelbordonado** [comment]
> One question how about the issues of not getting the password in the email? you remember for us we had to edit student password in the database manually

I had dig into it, I think I need to configure the EMAIL server in staging to get the emails. 
They were using mailtraip.io for testing.

I will get it done but we can approve this PR (is config not code)

**Comment by abelbordonado** [comment]
Approved as part of #4

**Review by kayiganwa04 — APPROVED** [review]
Approved

**Review by kayiganwa04 — APPROVED** [review]
Appoved

**Pull request merged**